app/views/guidance/dates_and_deadlines.html.erb

```diff
@@ -21,27 +21,28 @@
       <% table.with_body do |body| %>
         <% body.with_row do |row| %>
           <% row.with_cell do %>
-              <span class="no-wrap">11 October 2023</span>
+              <span class="no-wrap">11 October 2024</span>
```

Comment:
```suggestion
              <span class="no-wrap">9 October 2024</span>
```

That is the second Wednesday of October

Reply:
Thanks @defong . Fixed as part of the content changes from Jen in the following commit.